src/Sulu/Bundle/MediaBundle/Tests/Functional/Controller/MediaControllerTest.php

```diff
@@ -529,9 +552,6 @@ public function testCgetExcludedIds()
         $this->assertEquals(2, $response->total);
     }
 
-    /**
-     * Test GET all Media, without passing a locale.
```

Comment:
😢

Reply:
Does this add anything to `testCgetWithNoLocale`? IMO it does not. And most of the comments in here were not only redundant but simply wrong.

Reply:
no it does not add anything valuable 🙂